kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtNetworkServiceImpl.java

```diff
@@ -252,7 +252,7 @@
                                         } else {
                                             gwtNetConfig.setIpAddress("");
                                         }
-                                        if (addressConfig.getNetworkPrefixLength() >= 0
+                                        if (addressConfig.getNetworkPrefixLength() > 0
```

Comment:
@ibinshtok Do you remember why here we were allowing to pass the check also with 0? Because NetworkUtil.getNetmaskStringForm(addressConfig.getNetworkPrefixLength()) throws an exception if 0 is passed.